File: codegen_sources/test_generation/evosuite_tests_translators/tests/resources/expected_python_translations/java_list.py
import numpy as np
import math
from math import *
import collections
from collections import *
import heapq
import itertools
import random
import sys
import unittest


# TOFILL


class CLASS_db35bde703321c750c7134d5769b704c9ab7f9841c6654abb814683a361f9de1(
    unittest.TestCase
):
    def test0(self):
        arrayList0 = []
        double0 = 0.0
        arrayList0.append(double0)
        double1 = 1.0
        f_filled(arrayList0, double1)
        assert not (double1 in arrayList0)

    def test1(self):
        arrayList0 = []
        double0 = 0.0
        arrayList0.append(double0)
        f_filled(arrayList0, double0)
        assert 0.0 in arrayList0

    def test2(self):
        arrayList0 = []
        double0 = 9000.554
        arrayList0.append(double0)
        f_filled(arrayList0, double0)
        assert 9001 == arrayList0.size()

    def test3(self):
        arrayList0 = []
        double0 = 0.0
        f_filled(arrayList0, double0)
        assert not (double0 in arrayList0)


if __name__ == "__main__":
    unittest.main()

File: codegen_sources/test_generation/evosuite_tests_translators/tests/resources/expected_cpp_translations/java_list.cpp
#include <iostream>
#include <cstdlib>
#include <string>
#include <vector>
#include <fstream>
#include <iomanip>
#include <bits/stdc++.h>
#include "gtest/gtest.h"
#include "gmock/gmock.h"

using namespace std;



//TOFILL





  TEST(EvoSuiteTest, test0){
      vector<double> arrayList0;
      double double0 = (0.0);
      arrayList0.add(double0);
      double double1 = (1.0);
      f_filled(arrayList0, double1);
      ASSERT_FALSE (arrayList0.contains(double1));
  }

  TEST(EvoSuiteTest, test1){
      vector<double> arrayList0;
      double double0 = (0.0);
      arrayList0.add(double0);
      f_filled(arrayList0, double0);
      ASSERT_TRUE (arrayList0.contains(0.0));
  }

  TEST(EvoSuiteTest, test2){
      vector<double> arrayList0;
      double double0 = (9000.554);
      arrayList0.add(double0);
      f_filled(arrayList0, double0);
      ASSERT_EQ (9001,  arrayList0.size());
  }

  TEST(EvoSuiteTest, test3){
      vector<double> arrayList0;
      double double0 = (0.0);
      f_filled(arrayList0, double0);
      ASSERT_FALSE (arrayList0.contains(double0));
  }



int main(int argc, char **argv) {
  ::testing::InitGoogleTest(&argc, argv);
  return RUN_ALL_TESTS();
}